trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -683,18 +685,20 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
             # quantities for the dictionary are combined correctly. 
                 comm = _get_comm(())
                 if comm.size > 1:
-                    obs_dict_fields = [column_density,tau_ray,delta_lambda,
+                    obs_dict_fields = [column_density,tau_ray,EW, delta_lambda,
                                        lambda_obs, thermal_b, thermal_width]
                     obs_dict_fields = [comm.mpi_allreduce(field,op="sum") for field in obs_dict_fields]
 
-                 # Update the line_observables_dict with values for this line
-                obs_dict = {"column_density":column_density,
+                # Update the line_observables_dict with values for this line
+                EW = np.sum(1-np.exp(-current_tau_field))*self.bin_width
+                obs_dict = {"column_density_ray":column_density,
```

Comment:
You're making a backwards incompatible change here by changing the `column_density` to go from being the column density array to now just being the sum of that array, and making a new `column_density_ray` keyword to match the old array.  This is generally a bad idea as it will break other people's scripts who are using this.  Furthermore, we don't really need to store the column density array as well as the sum of that array.  I think the user can take the sum for herself.  So can you change this back to how it was?